SimplyPluralApi.yml

```diff
@@ -1530,74 +1576,91 @@ paths:
         "404":
           description: NOT FOUND
 
-  /v1/chat/message/:id:
```

Comment:
Seperate endpoint documentations for one and the same endpoint is not ideal, especially since everything except the "id" is different. Instead of using a required id param, create a seperate optional id param and reference that as the schema for all POST endpoints that have optional IDs. Duplicate endpoints only makes the list of endpoints larger and causes less of an overview, and people won't understand why there's two endpoints one with and one without ID.

Reply:
I just looked at the API spec and it does appear that it's not possible in the spec. However, then keep the original implementation and keep the "Optional" specification in the description. That's still better than having all POST endpoints multiple times.